homeassistant/components/luftdaten/__init__.py

```diff
@@ -57,6 +63,10 @@
         "mdi:thought-bubble-outline",
         CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
     ],
+    SENSOR_LAEQ: ["LAeq", "mdi:waveform", "LAeq"],
+    SENSOR_LA_MIN: ["LAmin", "mdi:waveform", "LAmin"],
+    SENSOR_LA_MAX: ["LAmax", "mdi:waveform", "LAmax"],
+    SENSOR_RADIATION_CPM: ["cpm", "mdi:atom", "cpm"],
```

Comment:
The unit of measurements look like the name.

`cpm` seems to be the common unit for the radiation. The SI unit of the sound pressure level is dB. As we talking about A-rated values then it should be dB(A) if I'm not wrong.

Reply:
`LAeq`, `LAmin` and `LAmax` are not SI units.

Reply:
ok, thanks for letting me know. Please tell me if I can help in any way.

Reply:
The units of measurement needs to updated before we can merge it.

Reply:
@hbitter could you advise in this? We need your expertise.